MycroftAI/mycroft-core
# Mimic2 fallback overlaps sentences

## Description
Mimic2 fallback to mimic overlaps sentences because multiple Mimic objects are being created for each sentence. This happens in 2 cases
1. When connection to mimic2 is lost while speaking multiple sentences 
2. Select mimic2 as your TTS engine and reset the device. Utterances overlap on startup

## How to test

- Set Mimic2 as you TTS engine
- Reset your device and listen to utterances on startup

If you do not want to reset your device
- Hit the top button & select 'WIFI' 
- Do not setup the wifi and ask general questions to Mark1 (ex: what is the time, weather..)
- Look for the var/logs/mycroft/auido.log to verify the connection error & fallback
- The utterance "Use your mobile device or computer to connect to the wifi network 'MYCROFT'. Then
    enter the password 1 2 3 4 5 6 7 8" should **not overlap**


## Contributor license agreement signed?
CLA [Yes] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
